webapp/src/selectors.js

```diff
@@ -1,6 +1,8 @@
 import {id as pluginId} from './manifest';
+import reducer from './reducer';
 
-const getPluginState = (state) => state['plugins-' + pluginId] || {};
+const defaultState = reducer({}, {});
```

Comment:
Thoughts on this approach? If it makes sense, I wonder if we should fold back into the starter template?

Reply:
How would it fold back in to the starter template? It doesn't seem to handle anything regarding redux in the webapp portion

Reply:
Ah, indeed. I'd forgotten just how slimmed down it was. Not sure if the addition of Redux is in scope, but maybe as a holistic change.